packages/adminrouter/extra/src/nginx.windows.agent.conf

```diff
@@ -0,0 +1,18 @@
+include includes/main/windows-common.conf;
```

Comment:
I would see how these replace the original names more clearly with `windows` after, rather than before, the stub name, e.g. `common-windows.conf` rather than `windows-common.conf`.

Reply:
Done